### DISCUSSION TITLE
GET organization members via API doesn't return beyond the first page of members listed on website

### DISCUSSION THREAD

**Discussion by danicamarsden** [community/community#24827] [Apps, API and Webhooks]
<p>GET orgs/:org/members only returns the members up to the end of the first page of people listed on the website (e.g. the first 30)</p>

**Comment by seveas** [comment, marked as answer]
<p>The GitHub API <a href="https://developer.github.com/v3/#pagination" rel="nofollow noopener">paginates its results</a>, so you’ll need to follow the “next” links to get all the members.</p>
